frontend/src/App.jsx

```diff
@@ -1,22 +1,30 @@
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import "./global.css";
 import Header from "./components/header";
-
+import LandingPage from "./pages/landing";
 import Login from "./pages/login";
-import Home from "./pages/home";
 import Signup from "./pages/signup";
 
 function App() {
   return (
     <BrowserRouter>
-      <Header />
-      <main style={{ paddingTop: "48px" }}>
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/login" element={<Login />} />
-          <Route path="/signup" element={<Signup />} />
-        </Routes>
-      </main>
+      <Routes>
+        <Route 
+          path="*" 
+          element={
+            <>
+              <Header />
+              <main style={{ paddingTop: "48px" }}>
+                <Routes>
+                  <Route path="/" element={<LandingPage />} />
+                  <Route path="/login" element={<Login />} />
+                  <Route path="/signup" element={<Signup />} />
+                </Routes>
+              </main>
+            </>
+          } 
+        />
+      </Routes>
```

Comment:
```suggestion
              <Header />
              <main style={{ paddingTop: "48px" }}>
                <Routes>
                  <Route path="/" element={<LandingPage />} />
                  <Route path="/login" element={<Login />} />
                  <Route path="/signup" element={<Signup />} />
                </Routes>
              </main>
```

We don't need to have headers on all the pages, including the route with random URL.